opsdroid/connector/gitlab/__init__.py

```diff
@@ -0,0 +1,333 @@
+"""A connector for Gitlab."""
```

Comment:
Let's rename this file to `gitlab.py` and create a new `__init__.py` file that just imports the things we want to expose. 

I set a bad president with having code in the `__init__.py` file (I didn't know any better when I started Opsdroid 😂) and we should try and clean that up.

Reply:
Haha that's a good point, I've been pretty bad at breaking this convention :laughing: